.github/workflows/tests.yml

```diff
@@ -154,6 +154,7 @@ jobs:
           # Build from unpacked dist tarball.
           cd /home/runner/work/lxd/lxd-test
           make deps
+          git -C "/home/runner/work/lxd/lxd-test/vendor/dqlite" log
```

Comment:
The `make deps` found a `DQLITE_PATH` but it was not a git repo so it kept going without running your `git checkout` of the specific commit.

Reply:
The cache is working against us here: https://github.com/canonical/lxd/actions/runs/15469677712/job/43550527186?pr=15729#step:16:25